README.md

```diff
@@ -272,6 +272,62 @@ spec:
       ipsecConfig: {}
 ```
 
+#### Configuring IPsec with OVNKubernetes at runtime
+OVN Kubernetes supports IPsec encryption dynamic enablement/disablement at runtime.
+IPsec protocol adds an ESP hdr to the tenant traffic, this additional hdr will store the security data that is needed by the traffic
+ends point to encrypt/decrypt the tenant traffic.
+
+In order to get IPsec to function properly, the cluster MTU size must be adjusted to fit the additional ESP hdr which added 46 bytes to the traffic,
```

Comment:
How about

`In order for IPsec to function properly the cluster MTU size must be decreased by 46 bytes to fit the additional ESP header added to each packet. This adjustment is not currently automatic and must be performed by the cluster administrator before enabling IPsec at runtime.`